src/writeData/components/WriteDataHelperBuckets.tsx

```diff
@@ -68,17 +70,29 @@ const WriteDataHelperBuckets: FC<Props> = ({
     body = (
       <List
         backgroundColor={InfluxColors.Grey5}
-        style={{height: '200px'}}
+        style={
+          disabled
+            ? {
+                pointerEvents: 'none',
+                height: '200px',
+                cursor: 'not-allowed',
+                borderColor: '#1a1a2a',
+                backgroundColor: '#1a1a2a',
```

Comment:
In general, we shouldn't use hex color literals in ui code, because these colors are defined by Clockface.

In a component, you'll want to import `InfluxColors` from clockface and use `InfluxColors.Grey15`, and in a stylesheet, you'd want to `@import '@influxdata/clockface/dist/variables.scss';` and use `$cf-grey-15`.

Reply:
gotcha, I'll remember that going forward, I'm going to remove this styling anyway and rely on the disabled styling that's already there.